alf/environments/suite_robotics.py

```diff
@@ -43,19 +43,42 @@ def is_available():
     return mujoco_py is not None
 
 
+@alf.configurable
 class SparseReward(gym.Wrapper):
     """Convert the original :math:`-1/0` rewards to :math:`0/1`.
     """
 
-    def __init__(self, env):
+    def __init__(self,
+                 env,
+                 reward_cap=1.,
+                 positive_reward=True,
+                 append_reward_dim=False):
```

Comment:
Docstrings are needed for the new arguments. The name of `reward_cap` doesn't correctly reflect its usage which is actually a scaling factor. Why do you need `append_reward_dim` (it should be put in another wrapper if needed)?